Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -26,9 +35,10 @@ Before sending your code to a VSP team for review, your code should meet all of
     * **PR title** should be a concise summary of the change.
     * In your **PR description**, discuss how code changes are related to the pull request objectives.
     * Reference the **GitHub issues** your PR relates to, in order to provide reviewers more context.
+    * If the pull request is a dependency/package upgrade or database migration for another pull request please reference that pull request.
```

Comment:
This _sounds_ right, but I'm not familiar with the process myself.

@annaswims What're your thoughts?

Reply:
I think this is generally covered in `discuss how code changes are related to the pull request objectives`

Reply:
removed